app/views/admin/users/index.html.erb

```diff
@@ -19,3 +19,26 @@
 </div>
 
 <%= link_to "New user", new_admin_user_path %>
+
+<div id="users">
+  <% @users.each do |user| %>
+    <table class="table table-striped table-bordered table-sm" style="background-color: #dee2e6;">
```

Comment:
@lime666 don't write styles on views. Please move this style to the `users.scss` file.